packages/common/stateInfo.ts

```diff
@@ -47,6 +47,7 @@ export interface BaseInfo extends Locale {
   signature?: string
   signatureType?: SignatureType
   contact: ContactData
```

Comment:
Sorry, let's just have the field
```
{
  alloy: { status: AlloyResponse | 'Error', id: string }
}
```

and save both the status and the id (sorry, I thought we were saving the status already).